### ISSUE TITLE
Files are not replaced by default when using multiple generic packs

### ISSUE THREAD

**Issue by ChipWolf** [itzg/docker-minecraft-server#1441]
### Describe the problem

On start with two generic packs, if the second generic pack contains a duplicate file, an interactive prompt to replace the file is presented. No is defaulted due to the lack of a tty.

### Container definition

`itzg/minecraft-server:2022.4.1-java8-multiarch`

### Container logs

```bash
chmod: changing permissions of '/data': Operation not permitted
[init] Running as uid=1000 gid=1000 with /data as 'drwxrwsr-x 16 0 2000 4096 Mar 22 18:12 /data'
[init] Resolved version given 1.12.2 into 1.12.2
[init] Resolving type given FORGE
[init] Checking Forge version information.
[init] Downloading generic pack from https://example.com/packs/forge-base-v10.4.0+en.zip
[init] Downloading generic pack from https://example.com/packs/forge-includes-v1.1.1.zip
[init] Checking if generic packs are up to date
[init] Generic pack(s) are out of date. Re-applying...
replace /tmp/generic_pack_base/version.txt? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)

unable to retrieve container logs for containerd://6a593c510ac5ff365ceae84f01382ab1b7fa72821f1a3dc7215fcf79445e66ba
```

**Comment by itzg** [comment]
For posterity (from Discord convo): this should be fixed by [this commit](https://github.com/itzg/docker-minecraft-server/commit/5ae5733741a7d5dc25656a8c0070476b5fdba5c5) included in 2022.5.0.

**Comment by ChipWolf** [comment]
Appears to be resolved. Thanks for the quick cutting of a new tag. 🤝
